ktor-server/ktor-server-plugins/ktor-server-swagger/jvm/src/io/ktor/server/swagger/Swagger.kt

```diff
@@ -0,0 +1,75 @@
+package io.ktor.server.swagger
+
+import io.ktor.server.application.*
+import io.ktor.server.html.*
+import io.ktor.server.request.*
+import io.ktor.server.response.*
+import io.ktor.server.routing.*
+import kotlinx.html.*
+import java.io.*
+
+/**
+ * Creates a `get` endpoint with [SwaggerUI] at [path] rendered from the OpenAPI file located at [swaggerFile].
+ */
+public fun Routing.swaggerUI(path: String, swaggerFile: String, block: SwaggerConfig.() -> Unit = {}) {
```

Comment:
what about loading from resources? Or may be just from InputStream/ByteReadChannel?
Also looks like this plugin is overall possible to use from native server (nix), so may be add support for it also

Reply:
Agree on support serving from resources and make it multiplatform.

Reply:
Currently we have no way to serve API file in native